src/Teams/Controllers/ManageSprintsController.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Teams.Models;
+using Teams.Services;
+
+namespace Teams.Controllers
+{
+    public class ManageSprintsController : Controller
+    {
+        private readonly IManageSprintsService _manageSprintsService;
+        private readonly IAccessCheckService _accessCheckService;
+        private readonly IManageTeamsService _manageTeamsService;
+
+
+        public ManageSprintsController(IManageSprintsService manageSprintsService, IAccessCheckService accessCheckService, IManageTeamsService manageTeamsService)
+        {
+            _manageSprintsService = manageSprintsService;
+            _manageTeamsService = manageTeamsService;
+            _accessCheckService = accessCheckService;
+        }
+
+        [Authorize, NonAction]
+        public async Task<List<Sprint>> GetAllSprints(int team_id, DisplayOptions options)
```

Comment:
Is there any reason why this action is public are we calling it from the client? If no, then it should be a private method or more preferable to hide this logic inside service layer